### PR TITLE
[PR] The proj files have been updated to enable SourceLink

### PR DESCRIPTION
**Pull request by JTOne123**
CSProj files have been updated to enable SourceLink in your nuget
---

*[This pull request was created with an automated workflow]*

I noticed that your repository and Nuget package are important for our .NET community, but you still haven't enabled SourceLink.

**We have to take 2 steps:**
1) Please approve this pull request and make .NET a better place for .NET developers and their debugging.
2) **Then just upload the .snupkg file** to https://www.nuget.org/ (now you can find the snupkg file along with the .nuget file)

You can find more information about SourceLine at the following links  
https://github.com/dotnet/sourcelink
https://www.hanselman.com/blog/ExploringNETCoresSourceLinkSteppingIntoTheSourceCodeOfNuGetPackagesYouDontOwn.aspx

If you are interesting about this automated workflow and how it works  
https://github.com/JTOne123/GitHubMassUpdater

*If you notice any flaws, please comment and I will try to make fixes manually*

### CODE REVIEW

**Comment by JohanLarsson** [comment]
I'm not opposed to merging this but out of curiosity how is SourceLink useful for an analyzer?

File: AspNetCoreAnalyzers.Vsix/AspNetCoreAnalyzers.Vsix.csproj [hunk 1]
```diff
@@ -7,8 +7,16 @@
     <RootNamespace>AspNetCoreAnalyzers</RootNamespace>
     <TargetName>AspNetCoreAnalyzers</TargetName>
     <NoWarn>NU1605</NoWarn>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
```
Comment (JohanLarsson) [inline AspNetCoreAnalyzers.Vsix/AspNetCoreAnalyzers.Vsix.csproj:16]: This repository uses paket to manage dependencies hence the build error on CI. This is not a problem for merging, I can merge it and fix after np.

**Comment by JTOne123** [comment]
it better to add this package through the paket

but I don't know if it makes sense for your nuget


Could the user make use of class from your lib?

**Comment by JohanLarsson** [comment]
> Could the user make use of class from your lib?
Using the code as a library is theoretically possible but I usually write analyzers so that everything is `internal`. Also not sure if there is a straightforward way to use it like this when packed like an analyzer.

**Comment by JTOne123** [comment]
agree

sorry for a mismatch, I've closed my PR

**Pull request closed without merging**